components/script/webdriver_handlers.rs

```diff
@@ -529,12 +529,16 @@ pub(crate) fn handle_execute_script(
                 Err(WebDriverJSError::JSError)
             };
 
-            reply.send(result).unwrap();
+            reply.send(result).unwrap_or_else(|_| {
```

Comment:
I think using an if with is_err() would be more clear than unwrapping the result.